code/modules/antagonists/fugitive/fugitive_outfits.dm

```diff
@@ -0,0 +1,75 @@
+/datum/outfit/prisoner
+	name = "Prison Escapee"
+	uniform = /obj/item/clothing/under/rank/prisoner
+	shoes = /obj/item/clothing/shoes/sneakers/orange
+	r_pocket = /obj/item/kitchen/knife/carrotshiv
+
+/datum/outfit/prisoner/post_equip(mob/living/carbon/human/H, visualsOnly=FALSE)
+	if(visualsOnly)
+		return
+	H.fully_replace_character_name(null,"NTP #CC-0[rand(111,999)]") //same as the lavaland prisoner transport, but this time they are from CC, or CentCom
+
+/datum/outfit/yalp_cultist
+	name = "Cultist of Yalp Elor"
+	uniform = /obj/item/clothing/under/rank/chaplain
+	suit = /obj/item/clothing/suit/holidaypriest
+	gloves = /obj/item/clothing/gloves/color/red
+	shoes = /obj/item/clothing/shoes/sneakers/black
+	mask = /obj/item/clothing/mask/gas/tiki_mask/yalp_elor
+
+/datum/outfit/waldo
+	name = "Waldo"
+	uniform = /obj/item/clothing/under/pants/jeans
+	suit = /obj/item/clothing/suit/striped_sweater
+	head = /obj/item/clothing/head/beanie/waldo
+	shoes = /obj/item/clothing/shoes/sneakers/brown
+	ears = /obj/item/radio/headset
+	glasses = /obj/item/clothing/glasses/regular/circle
+
+/datum/outfit/waldo/post_equip(mob/living/carbon/human/H, visualsOnly=FALSE)
+	if(visualsOnly)
+		return
+	H.fully_replace_character_name(null,"Waldo")
+	H.eye_color = "000"
+	H.gender = MALE
+	H.skin_tone = "caucasian3"
+	H.hair_style = "Business Hair 3"
+	H.facial_hair_style = "Shaved"
+	H.hair_color = "000"
+	H.facial_hair_color = H.hair_color
+	if(H.mind)
+		H.mind.AddSpell(new /obj/effect/proc_holder/spell/aoe_turf/knock(null))
+	var/list/no_drops = list()
+	no_drops += H.get_item_by_slot(SLOT_SHOES)
+	no_drops += H.get_item_by_slot(SLOT_W_UNIFORM)
+	no_drops += H.get_item_by_slot(SLOT_EARS)
```

Comment:
I wonder if fixing the radio on Wally is a good thing. The player is already alone, so if he can get a sec radio for example he should be able to switch to know if everyone is looking for him. I'm ok with all the rest tho

Reply:
Sure

Reply:
if he finds a radio technically he can just screwdrive the keys out but this is nbd either way imo

Reply:
Exactly my thoughts like yeah he could key but also why not

Reply:
Exactly my thoughts like yeah he could key but also why not